flycheck.el

```diff
@@ -7051,6 +7052,16 @@ See URL `http://sass-lang.com'."
           line-end))
   :modes scss-mode)
 
+(flycheck-define-checker scss-lint
+  "A SCSS syntax checker using SCSS-Lint.
+
+See URL `https://github.com/causes/scss-lint'."
+  :command ("scss-lint" "--formatter" "default" source)
```

Comment:
Right, that's definitely new since I last looked at scss-lint
On Sun, 5 Apr 2015 at 8:19 am, Sebastian Wiesner <notifications@github.com>
wrote:

> In flycheck.el
> <https://github.com/flycheck/flycheck/pull/598#discussion_r27775436>:
>
> > @@ -7051,6 +7052,16 @@ See URL `http://sass-lang.com'."
> >            line-end))
> >    :modes scss-mode)
> >
> > +(flycheck-define-checker scss-lint
> > +  "A SCSS syntax checker using SCSS-Lint.
> > +
> > +See URL `https://github.com/causes/scss-lint'."
> > +  :command ("scss-lint" "--formatter" "default" source)
>
> Just FYI, the option appears to be --format now, and the URL has changed
> to https://github.com/brigade/scss-lint
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/flycheck/flycheck/pull/598/files#r27775436>.
>